recipe/meta.yaml

```diff
@@ -59,3 +71,5 @@ about:
 extra:
   recipe-maintainers:
     - isuruf
+    - tadeu
+    - doxygen
```

Comment:
@doxygen Are you ok being added as a co-maintainer of this conda-forge package? This will give you read/write/merge privileges on the resulting feedstock repository.